packages/docs/.vuepress/components/ods-icon.vue

```diff
@@ -0,0 +1,29 @@
+<!-- Usage: <ods-icon icon="complete" /> -->
+
+<template>
+  <div class="ods-icon" v-html="require(`!html-loader!@okta/odyssey-icons/src/${icon}.svg`)" />
+</template>
+
+<style lang="scss">
+.ods-icon {
+  display: inline-block;
+
+  svg {
+    width: 24px;
+    height: 24px;
+    fill: currentColor;
+  }
+}
+</style>
+
+<script>
+export default {
+    name: 'ods-icon',
+    props: {
+        icon: {
+            type: String,
+            default: null
+        }
+    }
```

Comment:
style-nit: Looks like we have `4` spaces here while Vue traditionally follows `2` space indents.